pkg/platform/api/api.go

```diff
@@ -0,0 +1,201 @@
+package api
+
+import (
+	"context"
+	"encoding/json"
+	"github.com/bottlerocket-os/bottlerocket-update-operator/pkg/logging"
+	"github.com/pkg/errors"
+	"io/ioutil"
+	"net"
+	"net/http"
+	"time"
+)
```

Comment:
If you look at the import line in agent.go, you'll see that imports are separated into three sections:

1. standard library
2. First-party
3. Third-party

Can you do the same here?  Leaving a newline in between the sections will work properly with `goimports`.